libbeat/scripts/Makefile

```diff
@@ -213,7 +213,7 @@ system-tests-environment: prepare-tests build-image
 	${DOCKER_COMPOSE} run -e INTEGRATION_TESTS=1 -e TESTING_ENVIRONMENT=${TESTING_ENVIRONMENT} -e DOCKER_COMPOSE_PROJECT_NAME=${DOCKER_COMPOSE_PROJECT_NAME} beat make system-tests
 	#This is a hack to run x-pack/filebeat module tests
 	@XPACKBEAT="${ES_BEATS}/x-pack/${BEAT_NAME}" ; \
-	if [ -e "$$XPACKBEAT/tests/system" ] && [ $(XPACK_ONLY) = false ]; then \
+	if [ -e "$$XPACKBEAT/tests/system" ] && [ "$(BEAT_NAME)" != "libbeat" ] && [ $(XPACK_ONLY) = false ]; then \
```

Comment:
@andrewkroh FYI, I added this exception here

Reply:
Thanks for letting me know. I am working on a way to directly invoke the tests via `mage integTest` which well let us eventually clean up the Make logic.

Reply:
That PR is 
https://github.com/elastic/beats/pull/8956. FYI